src/bin/fish.rs

```diff
@@ -238,7 +236,7 @@
             retval = Ok(());
         } else {
             let backtrace = parser.get_backtrace(&cmd_wcs, &errors);
-            eprintf!("%s", backtrace);
+            eprint!("{backtrace}");
```

Comment:
note that `print!` locks stdout but `printf!` macros don't.
I don't think we need locking anywhere since we don't use threads very much but I guess it's harmless.